Tools/machines/greatlakes-umich/greatlakes_v100.sbatch

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash -l
+
+# Copyright 2024 The WarpX Community
+#
+# Author: Axel Huebl
+# License: BSD-3-Clause-LBNL
+
+#SBATCH -t 00:10:00
+#SBATCH -N 1
+#SBATCH -J WarpX
+#SBATCH -A <proj>
+#SBATCH --partition=gpu
+#SBATCH --exclusive
+#SBATCH --ntasks-per-node=2
+#SBATCH --cpus-per-task=20
+#SBATCH --gpus-per-task=v100:1
+#SBATCH --gpu-bind=single:1
+#SBATCH -o WarpX.o%j
```

Comment:
Recommend expliciting putting
`#SBATCH --mem=0`
to signify this request is allocating all the memory on the node. This should happen dynamically, since --exclusive is set but i think for users it is a good idea to put so they have some reference to what is implicitly happening with the job request

Reply:
Wait, `--mem=0` means all? o.0
I think `--exclusive` is a bit clearer for now.